ui/packages/catalog-ui-search/src/main/webapp/react-component/menu/menu.js

```diff
@@ -67,6 +67,12 @@ class Menu extends React.Component {
   render() {
     const { value, children } = this.props
 
+    if (typeof this.state.active === 'undefined' && children.length > 0) {
```

Comment:
Could this if else statement be just a ternary on children.length === 0? I think that could work if the menu doesn't render more than once in general. In the gazetteer, the component rerenders every time a letter is entered or deleted. Not sure if there are other cases where it would rerender.

Reply:
This logic has now been factored out into a method that will be called from different states of the React component, so if that condition wasn't necessary before, I'm afraid that it is now.